docs/source/mixed-precision.rst

```diff
@@ -3,39 +3,67 @@ Mixed precision training
 
 In this section we will describe
 
-* How to train existing models in mixed precision
-  (basically, two lines change in the config: dtype and loss scaling).
+* How to train models in mixed precisionfloat for training on Volta GPUs
 
-* How mixed precision is implemented in our toolkit (optimizer and regularizer
-  wrappers)
+* How to port existing models from float32 to mixed precisionfloat
+  
+* How mixed precision is implemented in our toolkit 
 
-* What are the general steps needed to add mixed precision support to a new
-  model. Should also contain example of what we did for our cases: profiling
-  GPU/CPU placement, profiling which operations are bottleneck,
-  debugging infs/nans, converting some layers to fp32, etc.
+Mixed precision training on Volta GPUs
+--------------------------
 
-Why mixed precision
--------------------
+Half-precision floating numbers (float16) have limited numerical range compared to single-precision float (float32). This constarined numerical range can lead to overflow or underflow during training. We use two techniques to prevent these hazards. First, we  maintain a "master" float32 copy of the weights that accumulates the gradients after each optimizer step.  Second, we use global scaling applied to the loss to prevent the loss of information during backpropagation. We demonstrated that this approach for GNMT(translation) and DeepSpeech2 (speech recognition).  Using this approach, we can reduce the memory consumption of deep learning models by nearly 2x. 
```

Comment:
Should we maybe start with description of what benefits MP brings (at least in theory)? Because if someone does not understand what it is, they will just see that we speak about problems and will not even realize why they need it (they might not even read to the last sentence).